.circleci/config.yml

```diff
@@ -60,7 +62,14 @@ workflows:
               only:
                 - master
                 - production
-
+      - deploy-gar:
```

Comment:
we probably need also `requires` like above, to make sure the docker image has been produced and cached